modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/loading/Load.scala

```diff
@@ -119,14 +120,9 @@ object Load {
       .limitRetries[LoaderAction[F, *]](MaxRetries)
       .join(RetryPolicies.exponentialBackoff(Backoff))
 
-  private def congratulate[F[_]: Monad: Logging: Timer](message: LoaderMessage.ShreddingComplete): LoaderAction[F, Unit] =
-    Timer[F].clock.instantNow.flatMap { now =>
-      val count = message.count.map(c => s"${c.good} events").getOrElse("volume is unknown")
-      val latency = message.timestamps.min match {
-        case Some(earliest) => s"${Duration.between(earliest, now).toSeconds} seconds"
-        case None => "unknown"
-      }
-      Logging[F].info(s"Folder [${message.base}] has been loaded and committed. Success!") *>
-        Logging[F].info(s"Folder [${message.base}] ($count) latency: $latency")
-    }.liftA
+  private def congratulate[F[_]: Logging: Monad](
+    loaded: LoaderMessage.ShreddingComplete
+  ): F[Unit] =
+    Logging[F].info(s"Folder ${loaded.base} loaded successfully") >>
```

Comment:
Just to re-iterate on what I said this morning - I think these logs were important not because they were showing numbers, but because they were human-readable and giving operator an understanding on what is going on in the pipeline. Just showing machine-readable strings would be the same.

Also, if we added something in 1.0.0 we should be careful about removing/changing it, unless we identified a problem with that format.